### DISCUSSION TITLE
What about the DNSBOMB?

### DISCUSSION THREAD

**Discussion by zbalkan** [TechnitiumSoftware/DnsServer#935] [General]
I recently came across this page mentioning a new attack called DNSBOMB
https://dnsbomb.net/

The page mentions two CVEs regarding the issue but the links are broken.

- CVE-2023-28456
- CVE-2023-49203

Though it requires IP spoofing and some owned domains for amplification and impacting only public domain servers, the article mentions that the authors got in contact. I wondered if it is considered to be fixed in the future or a WONTFIX issue.

**Comment by ShreyasZare** [comment]
Thanks for asking. This issue has been mitigated in v12.0 itself which you can read in the [change log](https://github.com/TechnitiumSoftware/DnsServer/blob/master/CHANGELOG.md#version-120).

**Reply by zbalkan** [reply]
I searched for the CVEs and the term DNSBOMB yet could not find anything. Thanks for the reference.

**Reply by ShreyasZare** [reply]
Yes, I have not updated the change log with details. Will get it done soon.
